### PR TITLE
LAV Filters Megamix 0.77.1-2

### PR DESCRIPTION
**Pull request by Witchilich**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Like PDFill, the manifest has to be deleted or updated with each update because theres only one url in the author's site that always directs to the latest version. No 32-bit version from 0.77.1-2 and hence no ReClock.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95771)

### CODE REVIEW

**Comment by Witchilich** [comment]
How is this different from https://github.com/microsoft/winget-pkgs/pull/1467 ?

**Comment by stephengillie** [comment]
I'm unable to tell if this is the correct domain for this package.
* https://github.com/microsoft/winget-pkgs/issues/95155

File: manifests/r/ranpha/LavFiltersMegamix/0.77.1-2/ranpha.LavFiltersMegamix.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PackageLocale: en-US
+Publisher: Hendrik Leppkes
+PackageUrl: https://anime.my/tutorials/watching-h264-videos-using-compute-unified-device-architecture-cuda/
+PublisherUrl: https://anime.my
+PublisherSupportUrl: https://anime.my/misc/contact-form/
+PrivacyUrl: https://anime.my/misc/privacy-policy/
+Author: ranpha
+PackageName: LAV Filters 0.77.1-2
+License: Freeware
+ShortDescription: LAV Filters Megamix is a codec pack of LAVFilters, ASSFilterMod and Madvr with video players like PotPlayer and MPC-HC.
```
Comment (stephengillie) [inline manifests/r/ranpha/LavFiltersMegamix/0.77.1-2/ranpha.LavFiltersMegamix.locale.en-US.yaml:12]:
```suggestion
ShortDescription: LAV Filters Megamix is a multiple codec pack with several popular video players included.
```

Comment (stephengillie) [reply manifests/r/ranpha/LavFiltersMegamix/0.77.1-2/ranpha.LavFiltersMegamix.locale.en-US.yaml:12]: This suggestion would clear the label `Policy-Test-1.2` by removing other product names.

**Comment by Witchilich** [comment]
> I'm unable to tell if this is the correct domain for this package.
> 
>     * [[Package Issue]: ranpha.LavFiltersMegamix Unofficial package released #95155](https://github.com/microsoft/winget-pkgs/issues/95155)

It's just a codec pack like K-Lite.
https://github.com/microsoft/winget-pkgs/tree/master/manifests/c/CodecGuide/K-LiteCodecPack
Also clsid2.mpc-hc contains LAV Filters
https://github.com/microsoft/winget-pkgs/tree/master/manifests/c/clsid2/mpc-hc
https://github.com/clsid2/mpc-hc
> The internal codecs are developed in an external project (LAV Filters). That project is still actively maintained, and MPC-HC will be updated periodically with the latest codecs. Same applies to code/files from other external projects that are used by MPC-HC, such as MediaInfo.

**Pull request closed without merging**